tests/spec/date_series_ops/test_date_comparisons.py

```diff
@@ -19,7 +17,7 @@
 def test_is_before(spec_test):
     spec_test(
         table_data,
-        p.d1.is_before(date(2000, 1, 1)),
+        p.d1.is_before("2000-01-01"),
```

Comment:
I think we should probably have a separate spec that demonstrates that we can take both types -- for documentation purposes.

Reply:
You're quite right, we should. I think I hesitated because I couldn't decide whether I ought to duplicate the entire suite with both types or not; so in the end I did nothing. Do you have a view?